workspaces/src/rpc/tool.rs

```diff
@@ -47,11 +50,44 @@ pub(crate) async fn url_create_account(
         .post(helper_addr)
         .header("Content-Type", "application/json")
         .body(serde_json::to_vec(&serde_json::json!({
-            "newAccountId": account_id.to_string(),
-            "newAccountPublicKey": pk.to_string(),
+            "newAccountId": account_id,
+            "newAccountPublicKey": pk,
         }))?)
         .send()
         .await?;
 
     Ok(())
 }
+
+pub(crate) fn write_cred_to_file(path: &Path, id: AccountId, sk: SecretKey) {
+    let mut file = File::create(path).expect("Failed to create / write a key file.");
+
+    #[cfg(unix)]
+    {
+        use std::os::unix::prelude::PermissionsExt;
+        let mut perm = file
+            .metadata()
+            .expect("Failed to retrieve key file metadata.")
+            .permissions();
+
+        #[cfg(target_os = "macos")]
```

Comment:
will this even be hit when in the block of `cfg(unix)` or does that only define what is compiling the code?

Reply:
macos is apart of the `unix` target_family so it should be hit. I noticed this because the macos tests were failing to compile for this case

Reply:
oh wow I completely blanked on that question, disregard :D

Reply:
ahh no worries